README.md

```diff
@@ -67,3 +67,6 @@ roslaunch aws_robomaker_small_warehouse_world view_small_warehouse.launch
 
 **Visit the [AWS RoboMaker website](https://aws.amazon.com/robomaker/) to learn more about building intelligent robotic applications with Amazon Web Services.**
 
+## Notes
+- Lighting might vary on different system(s) (e.g brighter on system without CPU and darker on system with GPU)
```

Comment:
Is this an ideal behavior?

Reply:
It is not ideal, but it is fix current broken lighting on CPU instance. We are gathering user feedback to consider proper fix for CPU/GPU rendering parity.